themes/default/content/templates/kubernetes/azure/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Kubernetes Cluster on Azure"
+layout: template
+
+# Make sure this is description accurate for this template.
+meta_desc: The Kubernetes Cluster template makes it easy to deploy a Kubernetes cluster on Azure with Pulumi and Azure Kubernetes Service (AKS).
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a Kubernetes cluster on Azure with Pulumi and Azure Kubernetes Service (AKS).
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: kubernetes-azure
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Azure
+  slug: azure
+
+# Be sure to replace this image. Figma source file:
+# https://www.figma.com/file/lGrSpwbGGmbixEuewMbtkh/Template-Architecture-Diagrams?node-id=15%3A196
+meta_image: meta.png
+
+# The content below is meant help you get started and to serve as a guide to work by. Feel free to adjust it needed for your template.
+---
+
+The Azure Kubernetes Cluster template creates an infrastructure as code project in your favorite language and deploys a managed Kubernetes cluster to Azure. The architecture includes an [Azure Virtual Network]({{< relref "registry/packages/azure-native/api-docs/network/virtualnetwork">}}) with three subnets (set to private by default) and deploys an [Azure Kubernetes Service (AKS) cluster]({{< relref "/registry/packages/azure-native/api-docs/containerservice/managedcluster" >}}) that provides a managed Kubernetes control plane. Kubernetes worker nodes are deployed on private subnets for improved security. The template generates a complete infrastructure as code program to give you a working project out of the box that you can customize easily and extend to suit your needs.
```

Comment:
Suggest changing "with three subnets (set to private by default)" to "with three subnets (for future scalability)" as Azure subnets don't have the same concept of public/private as AWS subnets do.

Also suggest changing "Kubernetes worker nodes are deployed on private subnets for improved security" to "Kubernetes worker nodes are deployed with private IP addresses for improved security and across multiple availability zones for improved resilience."